contracts/ArchetypeLogic.sol

```diff
@@ -124,8 +126,8 @@ struct BurnConfig {
   uint64 limit;
 }
 
-address constant PLATFORM = 0x86B82972282Dd22348374bC63fd21620F7ED847B;
-address constant BATCH = 0x6Bc558A6DC48dEfa0e7022713c23D65Ab26e4Fa7;
+address constant PLATFORM = 0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC;
```

Comment:
have to restore this. This gets search and replaced during tests during the `preTest` and `postTest` scripts.
Those only get triggered however if you run `npm run test`.